documentation/general/dash-sonic-hld.md

```diff
@@ -339,17 +339,20 @@ addresses                 = list of ip prefixes ',' separated. valid to have emp
 
 ```
 DASH_ACL_IN_TABLE:{{eni}}:{{stage}}
-    "acl_group_id": {{group_id}} 
+    "ipv4_acl_group_id": {{group_id}} (OPTIONAL)
```

Comment:
we use convention like this -` v4_meter_policy_id`. can you please change to `v4_acl_group_id`

Reply:
Done